tests/xfstests/install.pm

```diff
@@ -45,7 +45,7 @@ sub install_xfstests_from_repo {
     zypper_call('--gpg-auto-import-keys ref');
     record_info('repo info', script_output('zypper lr -U'));
     if (is_transactional) {
-        trup_call('pkg install xfstests fio');
+        trup_call('pkg install xfstests');
```

Comment:
Why not remove `fio` only for ALP testing for it with `is_alp` ?

Also, I would `record_soft_failure` not having `fio` in ALP.